lib/honeybadger/plugins/rails.rb

```diff
@@ -33,6 +33,22 @@ def render_exception(arg, exception)
         requirement { defined?(::Rails.application) && ::Rails.application }
 
         execution do
+          if ::Rails.application.config.exceptions_app.is_a?(::ActionDispatch::Routing::RouteSet)
+            Honeybadger.configure do |config|
+              config.before_notify do |notice|
+                begin
+                  original_path = notice.send(:opts)[:rack_env]["ORIGINAL_FULLPATH"]
```

Comment:
@fernandes hey one other question about this line -- why did you have to go through `notice.opts` (a private method) to get the internal data? On the original issue, @minhajuddin had given an example that seemed to indicate that `notice.url` would work:

https://github.com/honeybadger-io/honeybadger-ruby/issues/250#issuecomment-379780492

Just curious if there was a gotcha with that approach.

Reply:
hummmm I thought that Rails would only accept a `/path` for `Rails.application.routes.recognize_path` (given the method name), my mistake... a URL (in this case `notice.url`) works flawlessly

I'm pushing resolving with `notice.url`, good catch 👀 btw heheh